src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -0,0 +1,87 @@
+import Plugin from 'src/plugin-system/plugin.class';
+
+export default class CheckoutPlugin extends Plugin {
+
+    init() {
+        /* global adyenCheckoutConfiguration, AdyenCheckout */
+        /* eslint-disable no-unused-vars */
+
+        const paymentMethodTypeHandlers = {
+            'scheme': 'handler_adyen_cardspaymentmethodhandler'
```

Comment:
Create a constant for the formatted handler identifier with the value `handler_adyen_cardspaymentmethodhandler` inside this scope